public/locales/en/translation.json

```diff
@@ -221,7 +221,9 @@
   "retrocerts-certification": {
     "title": "Retroactive Certification for Unemployment",
     "question-page-title": "Retroactively Certify and Review",
+    "alert-found-save": "We found your information! Pick up where you last left off. Your previous weeks have been saved.",
```

Comment:
There's one other content update we'll need to make on the weeks-to-certify page. Right now, it says that progress isn't saved.